ipynb/Metadata cross correlation.ipynb

```diff
@@ -595,7 +573,9 @@
     "    if column in clean_up:\n",
     "        ordinal_replacement[column] = ordinal_assignment(exp, clean_up[column])\n",
     "    else:\n",
-    "        ordinal_replacement[column] = ordinal_assignment(exp)"
+    "        ordinal_replacement[column] = ordinal_assignment(exp)\n",
+    "    # SMJ: do the actual conversion of data. Make sure that case does not matter!\n",
+    "    map_[column] = map_[column].apply(lambda x: x.lower() if type(x) == str else x).replace(list(map(str.lower, ordinal_replacement[column].keys())), ordinal_replacement[column].values())"
```

Comment:
this is quite dense, would it be possible to refactor?

Reply:
I tried to make the code more explicit but found that it was a huge blow up + risk that dict order might get lost. Therefore, I decided to "just" add some comments to better explain what is going on. Is that OK with you?